.gitignore

```diff
@@ -1,4 +1,4 @@
-/target
+target
```

Comment:
@mareknovotny Is this change really necessary? Don't submodules already manage target folder ignoring?

Reply:
not all, it looks as If I run build there were few more directories as untracked files. It is anyway better to configure ignoring on top level too.